pom.xml

```diff
@@ -351,6 +351,10 @@
                     <groupId>log4j</groupId>
                     <artifactId>log4j</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
How did you determine the exclusions in the pom.xml?

Reply:
I looked at what changes were made to the financials POM for the 10-05 and 10-19 releases, and then added similar changes to our POM based on those. The exclusions here are based on those introduced with the two KualiCo patches for the related dependencies.

KualiCo also introduced the use of a new "duplicate-finder-maven-plugin", but that has not been included in this PR due to its potential impacts on our build. If you think we need to add that, I would suggest doing it in a separate user story.

Reply:
Thank you for explaining this Chad. The changes look good.